core/include/scipp/core/element/arithmetic.h

```diff
@@ -96,6 +99,10 @@ constexpr auto divide =
                  else
                    return a / b;
                }};
+constexpr auto mod =
+    overloaded{arg_list<int64_t, int32_t, std::tuple<int32_t, int64_t>,
+                        std::tuple<int64_t, int32_t>>,
+               [](const auto a, const auto b) { return a % b; }};
```

Comment:
This is not the same as Python's and numpy's % operator:
C++:
```.cpp
-1 % 2 == -1
```
Python:
```.py
-1 % 2 == 1
```

Reply:
😱